index.html

```diff
@@ -0,0 +1,76 @@
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css">
+    <title>Venice | Home</title>
+</head>
+<body>
+    <div class="grid-container">
+    <nav class="nav">
+        <div class="left">
+        <a href="#" class="logo"><img src="https://picsum.photos/id/1005/50/50"></a>
+        <a href="#" class="btn btn-link brand">Venice</a>
+        </div>
+        <div class="right">
+        <div class="input-group search"><input type="text" placeholder="Search"></div>
+        <button class="btn btn-primary">Login</button>
+        <div class="badge">
+            <a><i class="fas fa-heart"></i>
+            <div class="badge-icon">0</div></a>
+        </div>
+        <div class="badge cart-icon">
+            <a><i class="fas fa-shopping-cart"></i>
+            <div class="badge-icon">0</div></a>
+        </div>
+    </div>
+    </nav>
+    <div class="alert-container">Select your favourite color Theme and get all the products you vibe with😊</div>
+    <div class="themes">
+        <div class="theme-color theme-red">
+            <div class="theme-text">Red</div>
+        </div>
+        <div class="theme-color theme-orange">
+            <div class="theme-text">Orange</div>
+        </div>
+        <div class="theme-color theme-yellow">
+            <div class="theme-text">Yellow</div>
+        </div>
+        <div class="theme-color theme-green">
+            <div class="theme-text">Green</div>
+        </div>
+        <div class="theme-color theme-blue">
+            <div class="theme-text">Blue</div>
+        </div>
+        <div class="theme-color theme-indigo">
+            <div class="theme-text">Indigo</div>
+        </div>
+        <div class="theme-color theme-violet">
+            <div class="theme-text">Violet</div>
+        </div>
+    </div>
+    <!-- <div class="gallery">
+        <img src="https://images.bewakoof.com/uploads/grid/app/Mid-Size-Banner-Clogs-1640000276.jpg" alt="" srcset="">
+        <img src="https://images.bewakoof.com/uploads/grid/app/MidSizeBanner-Sliders-Flipflops-revised-01-1643895325.jpg" alt="" srcset="">
+    </div> -->
+    <!-- <div class="card card-horizontal-image-text">
+        <div class="card-body">
+          <h3 class="card-title">Summer Collection</h3>
+          <h5 class="card-subtitle">New Arrival</h5>
+          <div class="card-text">Check out our best Summer Collection to stay cool in style this season</div> -->
+          <!-- <div class="card-actions">
+          <a href="#" class="card-btn-1">READ</a>
+          <a href="#" class="card-btn-2">BOOKMARK</a>
+          <a href="#" class="card-icon-1"><i class="far fa-heart"></i></a>
+          <a href="#" class="card-icon-2"><i class="fas fa-share-alt"></i></a>
+          <a href="#" class="card-icon-3"><i class="fas fa-ellipsis-v"></i></a>
+        </div> -->
+        <!-- </div>
+        <img src="/Images/red-coat.jpg" class="card-img-1" alt="..."> -->
```

Comment:
commented code

Reply:
Thank you for pointing it out. i will remove it